cisco_secure_web_appliance/README.md

```diff
@@ -0,0 +1,278 @@
+    
+3. [Restart the Agent][3].
+
+### Configuration on Cisco Secure Web Appliance portal
+
+#### Steps to set time zone to GMT
+Datadog expects all the logs in GMT time zone by default. Please make sure that timezone configured in your Cisco Secure Web Appliance portal must be in GMT. Here are the steps to change the timzone:
+1. Go to **System Administration**, and then **Time Zone**.
+2. Click on **Edit Settings**.
+3. Select **GMT Offset** as the region.
+4. Select **GMT** as the country.
+5. Select **GMT (GMT)** as the time zone.
+6. Submit and commit the changes.
+
+#### Configure Log Subscriptions
+
+#### Configure Syslog Push for Access Logs:
+
+**Prerequisites:**
+- The datadog-agent server hostname where you want to push the logs.
+
+**Configuration:**
+
+1. Log in to Cisco Secure Web Appliance UI.
+2. Navigate to  **System Administration -> Log Subscriptions**.
+3. In order to add  Access Logs subscription, click on the **Add Log Subscription** button.
+4. Select Log Type as Access Logs.
+5. Provide Log Name.
+6. Choose the `Squid` option for Log Style.
+    **Note**: The default (squid) log style for access logs is supported. 
+7. Select `Syslog Push` option as the Retrieval Method.
+8. Provide Following Details.
+
+    Hostname: \<Datadog-Agent Host Server>
+
+    Port: \<Default Provided>
+
+    Protocol: UDP
+    
+    Maximum message size: \<Valid values for UDP are 1024 to 9216>
+    
+    Facility: \<Default Selected>
+9. Click on Submit. 
+10. Click on **Commit Changes** at the top right of the Log Subscriptions page.
+    **Note:** These changes will not go into effect until they are committed.
+
+#### Configure SCP on Remote Server for L4TM Logs
+
+**Prerequisites:**
+1. Requires hostname and username (admin account username is not necessary) of VM/machine on which datadog-agent is installed.
+
+**Configuration:**
+1. Navigate to  **System Administration -> Log Subscriptions** in Cisco Secure Web Appliance UI.
+2. To add a log subscription for Traffic Monitor Logs, click on  Add **Log Subscription**.  
+3. Select Traffic Monitor Logs as Log Type.
+4. Provide appropriate Log Name.
+5. For FileName, provide a new name or keep the default added name.
+6. Choose SCP on Remote Server as Retrieval Method.
+7. Provide Following Information.
+
+    SCP Host: \<SCP Host IP Address>
+
+    Directory: \<Path to Directory Where Logs would Get Stored>
+    **NOTE:** Make sure that Directory does not have any other log files.
+
+    SCP Port: \<Default Port>
+
+    Username: \<SCP Host Username>
+8. Click on Submit. After submitting, SSH key(s)  will get generated. Copy and save both the SSH key(s) as it is only visible once.
+9. Place the given SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
+10. Click on Commit Changes at top right-side of Log Subscriptions Page.
```

Comment:
```suggestion
8. Click **Submit**. After submitting, SSH key(s) are generated. Copy and save the SSH key(s) as it is only visible once.
9. Place the SSH key(s) into your `authorized_keys` file on the remote host so that the log files can be uploaded.
10. Click **Commit Changes** at top right of **Log Subscriptions** page.
```